deletefb/deletefb.py

```diff
@@ -51,9 +51,14 @@ def run_delete():
 
     args = parser.parse_args()
 
+    if args.password is not None or not args.password:
+        args_user_password = args.password
+    else:
```

Comment:
I think you could make this line cleaner by just having it
```
password = args.password or input('Enter your password:\n')
```

Reply:
There is a requirement for a check if the arg is empty. 
I could have written it as a oneliner, but I think for beginners it is more understandable.

Reply:
```args.password is not None or not args.password```
Wouldn't this always be true? Not sure if I'm just missing something lol

Reply:
I'd prefer either:
`args.password if args.password else input('Enter your password: ')` or `args.password or  input('Enter your password: ')`

Generally it's a good idea in Python to rely on the fact that certain values are "falsey" instead of checking for explicit values like None or False.

Reply:
But if I use `args.password or input('Enter your password: ')`, the "empty" would not be caught.

Reply:
@Spirit-act I'm not sure what you mean. I believe empty strings `''` evaluate to false?

Reply:
You can also do `args.password.strip()` if you're worried about whitespace, and it will evaluate to `False` if the string is made up of whitespace.

Reply:
unfortunately not, my tests:

> $ x = None
$ y = " "
$ a = x or input(': ')
: b
$ a = y or input(': ')
$

Reply:
```
>>> x = ""
>>> foo = x or input("blah: ")
blah: dsfsdfdsf
```

Reply:
I want to catch your workaround with space:

> A workaround at the moment is to prefix your invocation of the command with a space which will cause bash (and maybe other shells?) to not log it in your history.

But if you want, I'll change it to your preferences.

Reply:
I don't think we need to worry about that. It's specific to bash and wouldn't be caught by this.

Reply:
Ok, I have changed the code.